packages/ketcher-core/src/application/editor/data/monomers.ket

```diff
@@ -86965,6 +86974,706 @@
       }
     ],
     "naturalAnalogShort": "F"
+  },
+   "monomerTemplate-NMethylProline___NMethylProline": {
```

Comment:
Also my comment for keys naming and text formatting from CHEMs pull request applies here as well